besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -842,6 +843,12 @@ static class PrivacyOptionGroup {
       description = "Specifies the number of last blocks to cache  (default: ${DEFAULT-VALUE})")
   private final Integer numberOfblocksToCache = 0;
 
+  @Option(
+      names = {"--peertask-system-enabled"},
```

Comment:
should this be a --X hidden option since it's temporary?

Reply:
I didn't know we prefixed with X to denote hidden/temporary. I'll add it now